src/main/java/toothless/storage/Storage.java

```diff
@@ -0,0 +1,102 @@
+package toothless.storage;
+
+import toothless.exceptions.ToothlessExceptions;
+import toothless.task.Deadline;
+import toothless.task.Event;
+import toothless.task.Task;
+import toothless.task.ToDo;
+
+import java.io.*;
```

Comment:
Perhaps the individual classes should be imported explicitly instead of the whole library?